ci/no-std-check/Cargo.toml

```diff
@@ -6,7 +6,7 @@ resolver = "2"
 
 [dependencies]
 ibc = { path = "../../crates/ibc", default-features = false }
-ibc-proto = { version = "0.24.0", default-features = false }
+ibc-proto = { version = "0.24.0", default-features = false, features = ["parity-scale-codec", "borsh"]}
```

Comment:
`0.24.1` is now released with the required change; can you update it here and remove the `[patch.crates-io]` section?